lib/scripts/init_repo.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e #-x #ONLY FOR DEV
+
+repo_url=$1
+clone_from=${2:-$repo_url} #take 2nd arg if present otherwise repo_url
+
+#TODO remove sslverify
+git -c http.sslverify=false clone -l --no-hardlinks --no-checkout $clone_from . # ${pwd}
```

Comment:
remove `http.sslverify`! The only way how to avoid security issues is never write this temporary things.